src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -413,6 +415,9 @@ group target (CN is a reasonable default)
 
     private boolean disableRolePrefixing;
 
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "Diagnostic fields are left mutable so that groovy console can be used to dynamically turn/off probes.")
+    public static boolean GET_CN_FROM_ATTRIBUTES = Boolean.getBoolean(LDAPSecurityRealm.class.getName()+".getcnfromattributes");
```

Comment:
I would change the casing of the property to either `GET_CN_FROM_ATTRIBUTES` or `getCnFromAttributes`.

Reply:
done